fortex/health/processors/xray_processor.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
```

Comment:
You can follow [ICD Coding Processor](https://github.com/asyml/ForteHealth/blob/master/fortex/health/processors/icd_coding_processor.py) as a template to write this processor, since both these processors use pre trained Huggingface models, and it would make this processor more aligned with the coding patterns that we have been following for our source code.

Reply:
I have modified the code according to the above template.